src/Microsoft.DotNet.Build.Tasks/GetNetCoreAppVersionsFromFile.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GetNetCoreAppVersionsFromFile : Task
+    {
+        [Required]
+        public string PathToVersionsFile { get; set; }
+
+        [Output]
+        public string CorefxHash { get; set; }
```

Comment:
I expect we will want things like WCF as well when we do this for UAP package as an example and just making it more generic feels like the easier thing to do. 

Do you have a commit for how you are consuming this? Perhaps that will help inform me if it will be more difficult to consume in the generic fashion.